packages/terra-dialog/docs/README.md

```diff
@@ -13,9 +13,22 @@ Dialogs are temporary views that can be used in a myriad of ways. Dialogs have t
 import React from 'react';
 import Dialog from 'terra-dialog';
 
-<Dialog header="Header Content" footer="Footer Content">some body content</Dialog>
+const header = 'Header Content';
+
```

Comment:
It'd be great if we can get this API consistent. Ex: Since header doesn't require a wrapper, do we need them for body and footer?

Reply:
I suppose it depends on what you mean by "need." I didn't create one for header because ActionHeader is already the wrapper.

The wrappers were suggested by Brett, so I will discuss this with him when he gets in.

Reply:
I'm thinking put the wrappers inside the Dialog, and the content will just populate the wrappers in the dialog.

Reply:
Yeah, thinking on this some more that makes sense. For the dialog subcomponets, like`<Dialog.Body>`, these will switch to an internal implementation. Likely `<div className={dialogBodyClasses}>{dialogBody}</div>`

Reply:
https://github.com/cerner/terra-core/pull/1209/commits/bafa9ec603e96bfb692803bc6c334e0be2de9cea